src/Controller/Api/EntityController.php

```diff
@@ -237,15 +258,11 @@ private function processForm($formType, Entity $entity, Request $request, $metho
      * @param Request      $request     The request object.
      * @param integer|null $id          The id of the entity to validate against.
      *
-     * @access public
-     *
-     * @throws BadRequestHttpException When no property is supplied.
-     * @throws BadRequestHttpException When more than one property is supplied.
-     * @throws BadRequestHttpException The supplied property is not valid for the resource.
```

Comment:
These lines can be put back, when I initially did the phpcs fixing, it threw errors when having more than one @throws for the same named exception.

I think it's better if we give a description for each Exception. Wadda ya think?